sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -2346,6 +2347,45 @@ abstract class DDLSuite extends QueryTest with SQLTestUtils {
     }
   }
 
+  test("insert overwrite directory") {
```

Comment:
DDLSuite.scala is becoming bigger and bigger.

Move these two data source only test cases to `org.apache.spark.sql.sources.InsertSuite`?

Reply:
moved.